README.md

```diff
@@ -10,42 +10,82 @@ This repository contains the code for pre-processing structural T2-weighted, T2-
 3. Clinical Insight: Provide clinical insights into white matter changes, with magnetization transfer (MT) being particularly sensitive to white matter changes in non-compressed regions.
 4. Localized Evaluation: Provide spinal cord level-specific metrics to enable precise localization of spinal cord pathology.
 
-### About the OU Spine Dataset
-
+## Data collection and organization
+### OU Spine dataset
 The OU Spine dataset was acquired using the https://spine-generic.readthedocs.io. The study is ongoing, involving patients diagnosed with DCM and a control cohort of healthy subjects (HC). All MRI scans were acquired using a 3T MR750 GE scanner.
 Due to the ongoing nature of the study, patient data is still being collected and analyzed. However, sample patient and control data are available in the Example data folder. Full datasets are available upon reasonable request to the senior author.
 
 ### Data Format and Organization
 - All MRI datasets were converted from DICOM to NIFTI format and are organized following the Brain Imaging Data Structure (BIDS) format.
+  TODO: what was used for data conversion?
 - Spinal cord files are renamed according to BIDS standard https://bids.neuroimaging.io.
 
+Here is an example of the BIDS data structure: # TODO adjust with your dataset
```

Comment:
@Mfauziyya you could add something like this with your data, or refer ot your example data folder that has the right data organization